### ISSUE TITLE
Cannot Build on Windows

### ISSUE THREAD

**Issue by brianshen1990** [sunpy/ablog#12]
Windows 10
Python 2.7.14

Ablog 0.9.0, 0.9.1, tried them all and failed.
I even uninstalled Python and installed again, it is a pure environment and still failed. 

```
# Sphinx version: 1.6.7
# Python version: 2.7.14 (CPython)
# Docutils version: 0.14 
# Jinja2 version: 2.3
# Last messages:
#   
#   looking for now-outdated files...
#   none found
#   pickling environment...
#   done
#   checking consistency...
#   done
#   preparing documents...
#   done
#   writing output... [ 33%] about
# Loaded extensions:
#   ablog (0.9.0) from c:\python27\lib\site-packages\ablog\__init__.pyc
#   sphinx.ext.todo (1.6.7) from c:\python27\lib\site-packages\sphinx\ext\todo.pyc
#   sphinx.ext.intersphinx (1.6.7) from c:\python27\lib\site-packages\sphinx\ext\intersphinx.pyc
#   alabaster (0.7.10) from c:\python27\lib\site-packages\alabaster\__init__.pyc
#   sphinx.ext.extlinks (1.6.7) from c:\python27\lib\site-packages\sphinx\ext\extlinks.pyc
Traceback (most recent call last):
  File "c:\python27\lib\site-packages\sphinx\cmdline.py", line 306, in main
    app.build(opts.force_all, filenames)
  File "c:\python27\lib\site-packages\sphinx\application.py", line 339, in build
    self.builder.build_update()
  File "c:\python27\lib\site-packages\sphinx\builders\__init__.py", line 329, in build_update
    'out of date' % len(to_build))
  File "c:\python27\lib\site-packages\sphinx\builders\__init__.py", line 395, in build
    self.write(docnames, list(updated_docnames), method)
  File "c:\python27\lib\site-packages\sphinx\builders\__init__.py", line 432, in write
    self._write_serial(sorted(docnames))
  File "c:\python27\lib\site-packages\sphinx\builders\__init__.py", line 441, in _write_serial
    self.write_doc(docname, doctree)
  File "c:\python27\lib\site-packages\sphinx\builders\html.py", line 602, in write_doc
    self.handle_page(docname, ctx, event_arg=doctree)
  File "c:\python27\lib\site-packages\sphinx\builders\html.py", line 990, in handle_page
    output = self.templates.render(templatename, ctx)
  File "c:\python27\lib\site-packages\sphinx\jinja2glue.py", line 176, in render
    return self.environment.get_template(template).render(context)
  File "c:\python27\lib\site-packages\jinja2\environment.py", line 705, in render
    return self.environment.handle_exception(exc_info, True)
  File "c:\python27\lib\site-packages\ablog\templates\page.html", line 2, in top-level template code
    {% set fa = ablog.fontawesome %}
  File "c:\python27\lib\site-packages\alabaster\layout.html", line 1, in top-level template code
    {%- extends "basic/layout.html" %}
  File "c:\python27\lib\site-packages\sphinx\themes\basic\..\basic/layout.html", line 175, in top-level template code
    {%- block content %}
  File "c:\python27\lib\site-packages\alabaster\layout.html", line 40, in block "content"
    {{ super() }}
  File "c:\python27\lib\site-packages\jinja2\sandbox.py", line 259, in call
    return __context.call(__obj, *args, **kwargs)
  File "c:\python27\lib\site-packages\sphinx\themes\basic\..\basic/layout.html", line 193, in block "content"
    {%- block sidebar2 %}{{ sidebar() }}{% endblock %}
  File "c:\python27\lib\site-packages\sphinx\themes\basic\..\basic/layout.html", line 193, in block "sidebar2"
    {%- block sidebar2 %}{{ sidebar() }}{% endblock %}
  File "c:\python27\lib\site-packages\jinja2\sandbox.py", line 259, in call
    return __context.call(__obj, *args, **kwargs)
  File "c:\python27\lib\site-packages\sphinx\themes\basic\..\basic/layout.html", line 64, in template
    {%- include sidebartemplate %}
  File "c:\python27\lib\site-packages\ablog\templates\recentposts.html", line 6, in top-level template code
    <li><a href="{{ pathto(recent.docname) }}{{ anchor(recent) }}">{{ recent.date.strftime(gettext(ablog.post_date_format_short)) + " - " + recent.title }}</a></li>
  File "c:\python27\lib\site-packages\jinja2\sandbox.py", line 259, in call
    return __context.call(__obj, *args, **kwargs)
ValueError: Invalid format string
```

**Comment by nabobalis** [comment]
Thanks for this!

This is what I get for not setting up a Windows CI. I will set one up and try to bug fix this ASAP.

**Comment by nabobalis** [comment]
@brianshen1990 I pushed a fix as v0.9.2 to PyPi. Hopefully your issue should be fixed. Can you check for me please?

**Comment by brianshen1990** [comment]
I have tested it, it works OK now. Thanks~
